SasquatchMac/SasquatchMacObjC/AppDelegate.m

```diff
@@ -22,11 +21,24 @@ - (instancetype)init {
   [MSMobileCenter start:@"8649b73e-6df0-4985-a039-8ab1453d44f3"
            withServices:@[ [MSAnalytics class], [MSCrashes class], [MSPush class] ]];
   [MobileCenterProvider shared].mobileCenter = [[MobileCenterDelegateObjC alloc] init];
-  return self;
+  [self updateMobileCenterState];
```

Comment:
I am not very familiar with macOS. Is this a norm on macOS to configure libraries in the `applicationDidFinishLaunching` delegate? Otherwise, do we know where macOS devs are expecting libraries to be configured/started?